docs/control-plane.md

```diff
@@ -57,6 +55,8 @@ KFService offers a few high level specifications for common ML technologies. The
 | spec.[default,canary].scikitlearn | [ScikitLearn](#ScikitLearn) | A high level specification for ScikitLearn models. |
 | spec.[default,canary].pytorch     | [Pytorch](#Pytorch)         | A high level specification for Pytorch models. |
 | spec.[default,canary].custom      | [Custom](#Custom)           | A flexible custom specification for arbitrary customer provided containers. |
+| spec.[default,canary].minReplicas | Integer                     | An optional integer specifying the minimum scale of the default model definition. |
+| spec.[default,canary].maxReplicas | Integer                     | An optional integer specifying the maximum scale of the default model definition. |
```

Comment:
You missed the section:
### Replicas

Reply:
it is in this commit https://github.com/kubeflow/kfserving/pull/85/commits/22460f433d6142242be5b3a7c3554024e2054ffe, not sure why not reflected.